src/adagrams.js

```diff
@@ -1,8 +1,112 @@
-const Adagrams = {
+import { tsConstructorType } from "@babel/types";
+
+class Adagrams {
+
+  // Draw letters for a game of Adagrams.
   drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+    const constLetterPool = ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'B', 'B', 'C', 'C', 'D', 'D', 'D', 'D', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'F', 'F', 'G', 'G', 'G', 'H', 'H', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'J', 'K', 'L', 'L', 'L', 'L', 'M', 'M', 'N', 'N', 'N', 'N', 'N', 'N', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'P', 'P', 'Q', 'R', 'R', 'R', 'R', 'R', 'R', 'S', 'S', 'S', 'S', 'T', 'T', 'T', 'T', 'T', 'T', 'U', 'U', 'U', 'U', 'V', 'V', 'W', 'W', 'X', 'Y', 'Y', 'Z'];
+    let playerHand = [];
+
+    for (let i = 0; i < 10; i++) {
+      const randomNum = Math.floor(Math.random() * (constLetterPool.length + 1));
```

Comment:
Nice way to choose 10 random letters. However, what's the stop you from choosing 10 Z's, for instance.